citi-onboarding/lgbtqia
# Feature/history section

## What I did

- create history component and add it to App.tsx

## How to test

0. If you don't have this repository in your computer:
   1. create a folder
   2. In terminal
	   1. find your folder
	   2. `git clone https://github.com/citi-onboarding/lgbtqia.git`

1. Open the terminal and go to the folder of this project
2. In terminal:
   1. `git fetch`
   2. `git checkout Feature/HistorySection`
   3. open the client folder and enter `yarn install` and then `yarn start`
4. Check to see if there are any errors
5. Check if it looks ok in differently sized screens
7. Test the things that you think are worth testing, even the ones that are not in this description
